snippets/MappedList.md

```diff
@@ -0,0 +1,40 @@
+### MappedList
+
+Renders a list of elements from an array of data.
+
+Use the value of the `isOrderedList` prop to conditionally render a `<ol>` or `<ul>` list.
+Use `Array.prototype.map` to render every item in `data` as a `<li>` element and give it a `key`.
+`data` can either be an array of objects with the `id` and `value` properties or an array of primitives.
+Omit the `isOrderedList` prop to render a `<ul>` list by default.
+
+```jsx
+function MappedList(props) {
```

Comment:
Have we come up with a naming convention? Often you only see first letter capitalized if they're classes(an old fallback to OOP styling) and lowercase if they're functions so that they're easier to differentiate. Do we want to emulate that with our repo?